cpp/examples/ide_secir.cpp

```diff
@@ -35,14 +34,14 @@ int main()
     ScalarType tmax   = 10;
     ScalarType N      = 10000;
     ScalarType deaths = 13.10462213;
-    ScalarType dt     = 1;
+    ScalarType dt     = 0.01;
```

Comment:
maybe write a short comment, what you have learned form other choices of dt.

Reply:
The study of the order of convergence is not yet finished, so I don't think it's appropriate here.
The point was simply to allow different step sizes in this example. Or did you think of something else?